### DISCUSSION TITLE
Phase B voltage and Power way off

### DISCUSSION THREAD

**Discussion by zwizard** [emporia-vue-local/esphome#212] [General]
My Phase B voltage is 2.7 V and the power is a LOT lower than it should be. Any advice?
![image](https://github.com/emporia-vue-local/esphome/assets/24253773/9ad6537f-9e26-460d-acdf-837286a19bf1)

**Comment by flaviut** [comment]
You're going to need to open up the panel, keep in mind there's dangerous voltages in there.

1. Check all voltage connections are made tightly. I prefer Wago connectors since they're impossible to do wrong, but open up your wire nuts if you have them & make sure you're getting good contact.
2. Check the voltage using a multimeter in VAC mode between phase a / neutral, phase b / neutral. Does it match what you see here?

**Comment by zwizard** [comment]
Voltage on A & B is around 122.
I have a clamp on Amp meter and A is around 19 and B is around 17.
I guess I don't understand where the unit gets is B voltage from.

**Reply by flaviut** [reply]
Voltage doesn't come from the clamps, the voltage comes from the wiring harness. Are you matching the wire colors correctly between real life & the config?

![image](https://github.com/emporia-vue-local/esphome/assets/5213469/771b9428-2bcc-4963-a6fb-0dfb5625e0ec)

**Comment by zwizard** [comment]
I see. It would help if I read the directions. LOL
I saw a youtube video where a guy only used the black wire. All of the others went to neutral.

**Comment by zwizard** [comment]
Ok, I got that fixed but now no power reading.
![image](https://github.com/emporia-vue-local/esphome/assets/24253773/e3a6c842-cf7b-47f9-90b1-8c640b11f4dd)

**Reply by flaviut** [reply]
sounds like https://github.com/emporia-vue-local/esphome/discussions/127

**Comment by zwizard** [comment]
I swapped the clamp on B and that fixed it. Thanks so much!
